STAR-Delegation/draft-ietf-acme-star-delegation.md

```diff
@@ -205,18 +205,67 @@ phase completes successfully.
 ## Delegated Identity Profile
 {: #sec-profile}
 
-This section defines a profile of the ACME protocol, to be used between the NDC and IdO.
+This section defines a profile of the ACME protocol, to be used between the NDC
+and IdO.
+
+### Delegation Configuration
+{: #sec-profile-dele-config}
+
```

Comment:
Maybe clarify the context here, something like: An NDC identifies itself to the IdO as an ACME account. The IdO can delegate multiple names through each NDC, and these configurations are described through "delegation" objects associated with the NDC's Account object on the IdO.

Reply:
great, will do.